client/multi-currency/index.js

```diff
@@ -127,13 +129,68 @@ if ( storeSettingsSection ) {
 	)[ 0 ];
 	storeSettingsContentContainer.appendChild( storeSettingsContent );
 
+	// Place the Store Settings element inside of storeSettingsContainer
+	const storeSettingsNote = document.querySelector(
+		'#wcpay_currencies_settings_section > div > p'
+	);
+
+	if ( storeSettingsNote )
+		storeSettingsContentContainer.appendChild( storeSettingsNote );
+
 	// Position and style Submit button
 	submitButton.style.textAlign = 'right';
 	submitButton.style.display = 'flex';
 	submitButton.classList.add( 'wcpay-settings-layout' );
 	submitButton.classList.remove( 'submit' );
 
 	toggleSettingsSectionDisplay();
+
+	const previewPanePlaceholder = document.querySelector(
+		'#preview-pane-placeholder'
+	);
+
+	const getFirstEnabledCurrencyFromList = () => {
+		const currenciesList = enabledCurrenciesListItemsExceptPlaceholders();
+		if ( 1 < currenciesList.length ) {
+			return currenciesList
+				.slice( 1, 2 )[ 0 ]
+				.classList.toString()
+				.replace( 'enabled-currency ', '' )
+				.trim()
+				.toUpperCase();
+		}
+		return null;
+	};
+
+	const PreviewModalHelper = () => {
+		const [ isPreviewModalOpen, setPreviewModalOpen ] = useState( false );
+
+		/* eslint-disable jsx-a11y/anchor-is-valid */
+		return (
+			<>
+				<a
+					href="#"
+					onClick={ () => {
+						setPreviewModalOpen( true );
+					} }
+				>
+					{ __( 'Preview', 'woocommerce-payments' ) }
+				</a>
+				<PreviewModal
+					isPreviewModalOpen={ isPreviewModalOpen }
+					setPreviewModalOpen={ setPreviewModalOpen }
+					isAutomaticSwitchEnabledValue={ true }
+					isStorefrontSwitcherEnabledValue={ false }
+					clientCurrency={ getFirstEnabledCurrencyFromList() }
```

Comment:
Just leaving a comment here till the first currency approach is finalized or not.

Reply:
Updated the code on 4ffab70, removed all instances of `clientCurrency` and moved the simulation currency selection to PHP side. https://github.com/Automattic/woocommerce-payments/blob/4ffab7001b0e47f3107ecd4d5d7b60773bbf1f0b/includes/multi-currency/MultiCurrency.php#L1199-L1207